reactor-core/src/test/java/reactor/core/publisher/MonoZipTest.java

```diff
@@ -273,6 +275,45 @@ public void whenMonoJust6() {
 		            .verifyComplete();
 	}
 
+	@Test
```

Comment:
Could you remove the usage of `MonoProcessor` from these 4 tests? I don't think it is necessary, and since the `MonoProcessor` might get deprecated I'd rather avoid it in new tests.

Reply:
I've removed `MonoProcessor`. I also removed the `isError()`, `isSuccess()` and `isTerminated()` asserts, since I was under the impression that `verifyComplete()` already does this. Please let me know if this is not the case.

Reply:
yes this is correct 👍